docker-hub/index.md

```diff
@@ -98,119 +25,69 @@ GitHub and Bitbucket and push them to Docker Hub.
 
 Docker provides a [Docker Hub CLI](https://github.com/docker/hub-tool#readme){: target="_blank" rel="noopener" class="_"} tool (currently experimental) and an API that allows you to interact with Docker Hub. Browse through the [Docker Hub API](/docker-hub/api/latest/){: target="_blank" rel="noopener" class="_"} documentation to explore the supported endpoints.
 
-The following section contains step-by-step instructions on how to easily get started with Docker Hub.
+<div class="component-container">
+    <!--start row-->
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+        <div class="component">
+             <div class="component-icon">
+                 <a href="/docker-id/"><img src="/assets/images/laptop.svg" alt="Docker ID" width="70" height="70"></a>
+             </div>
+                 <h2 id="docker-id"><a href="/docker-id/">Create a Docker ID </a></h2>
+                <p>Sign up and create a new Docker ID</p>
+        </div>
+      </div>
+      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+        <div class="component">
+            <div class="component-icon">
+                 <a href="/docker-hub/repos/"><img src="/assets/images/explore.svg" alt="Docker Compose" width="70" height="70"></a>
+            </div>
+                <h2 id="docker-repos"><a href="/docker-hub/repos/">Create a repository</a></h2>
+                <p>Create a repository to share your images with your team, customers, or Docker community. </p>
+         </div>
+     </div>
+       <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+        <div class="component">
+            <div class="component-icon">
+                 <a href="/docker-hub/"><img src="/assets/images/checklist.svg" alt="quickstart" width="70" height="70"></a>
+            </div>
+                <h2 id="docker-hub"><a href="/docker-hub/">Quickstart</a></h2>
+                <p>Step-by-step instructions on getting started Docker Hub.</p>
```

Comment:
Sorry, I missed this in the first review. It seems like a word is missing between `getting started` and `Docker Hub`.  
```suggestion
                <p>Step-by-step instructions on getting started on Docker Hub.</p>
```